app/javascript/components/patient/history/EditHistoryModal.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { PropTypes } from 'prop-types';
+import { Button, ListGroup, Modal } from 'react-bootstrap';
+import moment from 'moment';
+import { formatRelativePast } from '../../../utils/DateTime';
+
+class EditHistoryModal extends React.Component {
+  render() {
+    return (
+      <Modal size="lg" show centered onHide={this.props.toggle}>
+        <Modal.Header closeButton>
+          <Modal.Title>Comment History</Modal.Title>
+        </Modal.Header>
+        <Modal.Body>
+          <ListGroup>
+            {this.props.versions
+              .sort((a, b) => {
+                return moment.utc(b.created_at).diff(moment.utc(a.created_at));
+              })
+              .map(version => (
+                <ListGroup.Item key={version.id}>
+                  <div className="text-muted mb-1">
+                    {formatRelativePast(version.created_at)} by {version.created_by}
```

Comment:
this is probably more of a question/discussion for the PO team, but should we include the absolute timestamp somehow as well?

Reply:
honestly, it looked way too cluttered with both.  will run it by them though

Reply:
only keeping the relative date, but did tweak the look on some feedback which I just pushed